Astronomy/Astronomy/Helper Files/FetchPhotoOperation.m

```diff
@@ -0,0 +1,85 @@
+//
+//  FetchPhotoOperation.m
+//  Astronomy
+//
+//  Created by Karen Rodriguez on 5/19/20.
+//  Copyright © 2020 Wyatt Harrell. All rights reserved.
+//
+
+#import "FetchPhotoOperation.h"
+#import "Astronomy-Swift.h"
+
+@interface FetchPhotoOperation()
+
+@property BOOL executing;
+@property BOOL finished;
+//@property BOOL cancelled;
+@property BOOL asynchronous;
+
+@end
+
+@implementation FetchPhotoOperation
+
+@synthesize executing = _executing;
+@synthesize finished = _finished;
+//@synthesize cancelled = _cancelled;
+@synthesize asynchronous = _asynchronous;
+
+- (instancetype)initWithReference:(Photo *)photo {
+    self = [super init];
+    if (self) {
+        _photoReference = photo;
+        _imageData = nil;
+        _asynchronous = YES;
+    }
+    return self;
+}
+
+- (void)start {
+    // Tell observers we about to change this
+    [self willChangeValueForKey:@"isExecuting"];
+    self.executing = YES;
+    // Let observers know we changed this
+    [self didChangeValueForKey:@"isExecuting"];
```

Comment:
This repeated code would be good to put in a computed property for `self.executing`.